server/main.go

```diff
@@ -5,9 +5,19 @@ import (
 	"net/http"
 )
 
-func main() {
+func listenerFunc() {
```

Comment:
why replace `main` with `listenerFunc()`.. This is breaking change for server. IMO

Reply:
same as above